src/ussd_registration.js

```diff
@@ -1225,7 +1225,7 @@ go.app = (function () {
     self.add('state_clinic_date_reminders_optout', function (name) {
       var contact = self.im.user.answers.contact;
       var text = $([
-        "Based on what you told me, I think your next clinic visit is {{next_clinic_visit}}"
+        "Based on what you told me, I think your next clinic visit is {{next_clinic_visit.split('T')[0]}}."
```

Comment:
Could we rather do this in the `context(...)` call? 

I'm not sure if there are translations for this, but the issue is if we send this to translators, they will have to keep the exact `{{next_clinic_visit.split('T')[0]}}` in their translation, and if they change anything it will break, so I think it's better to have a simpler `{{next_clinic_visit}}` in the copy.

Reply:
That's true. Fixing this